### PR TITLE
Id generator

### PR DESCRIPTION
**Pull request by krtrinkunas**
add class IDGenerator

### CODE REVIEW

File: MapApp/EncounterMe/Functions/IDGenerator.cs [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        private bool idIsSet = false;
+
+        private Hashtable locations = new Hashtable();
+        public void setID (List<Location> loc)
+        {
+            if (loc != null)
+            {
+                foreach (Location location in loc)
+                {
+                    Console.WriteLine(location.ID + " " + location.Name);
```
Comment (DominykasVi) [inline MapApp/EncounterMe/Functions/IDGenerator.cs:31]: Should we leave this in prod?

**Review by DominykasVi — APPROVED** [review]
Overall looks good

**Review by AdamBeresnev — APPROVED** [review]
Looks good

**Comment by EmilijaSvirsk** [comment]
Can merge

**Comment by krtrinkunas** [comment]
add class IDGenerator

**Pull request merged**